src/app/application.cpp

```diff
@@ -463,13 +463,22 @@ void Application::sendNotificationEmail(const BitTorrent::Torrent *torrent)
                      content);
 }
 
+void Application::torrentAdded(BitTorrent::Torrent *const torrent)
```

Comment:
```suggestion
void Application::torrentAdded(const BitTorrent::Torrent *torrent)
```

Reply:
Same as below: even I updated, but now the two event handlers' signatures diverge.